deep_daze/deep_daze.py

```diff
@@ -308,7 +314,13 @@ def __init__(
             self.epochs = 1 + (num_words - self.num_start_words) / self.words_per_epoch
             # add one epoch if not divisible
             self.epochs = int(self.epochs) if int(self.epochs) == self.epochs else int(self.epochs) + 1
-            print("Running for ", self.epochs, "epochs")
+            if self.separator is not None:
+                if self.separator not in text:
+                    print("Separator '"+self.separator+"' will be ignored since not in text!")
+                    self.separator = None
+                else:
+                    self.epochs = len(list(filter(None,text.split(self.separator))))
+            print("Running for", self.epochs, "epochs" + (" (splitted with '"+self.separator+"' as the separator)" if self.separator is not None else ""))
```

Comment:
recommend changing "splitted" to "split"

Reply:
Oh, you're right